main/lsp/LSPTask.cc

```diff
@@ -293,8 +293,8 @@ LSPQueryResult LSPTask::queryBySymbol(LSPTypecheckerDelegate &typechecker, core:
         const auto &usedSymbolNames = hash.usages.symbols;
         auto ref = core::FileRef(i);
 
-        const bool fileIsValid = ref.exists() && (ref.data(gs).sourceType == core::File::Type::Normal ||
-                                                  ref.data(gs).sourceType == core::File::Type::Package);
+        const bool fileIsValid =
+            ref.exists() && (ref.data(gs).sourceType == core::File::Type::Normal || ref.data(gs).isPackage());
```

Comment:
Packages will always be `File::Type::Normal` now, right? Is there ever a situation in which the second half of this or will get evaluated?

Reply:
Yeah, that's a really good point. I'll remove the `isPackage` call, as it's not contributing anything, and the intent of the previous check was that this wasn't any of the other cases for that enum.